app/controllers/institutions_controller.rb

```diff
@@ -32,7 +32,7 @@ def new
   end
 
   def create
-    unless PendingInstitutionInvite.where(invited_user_id: current_user,  status: 'pending').count > 0
+    if PendingInstitutionInvite.where(invited_user_id: current_user,  status: 'accepted').count > 0
```

Comment:
**Important** 
This means: a user that already was invited to create a new institution (and accepted it) cannot be invited again to create another institution